File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    answer = 0
    can=["aya", "ye", "woo", "ma"]
    for b in babbling:
        word=b
        word=word.replace(can[0],'1')
        word=word.replace(can[1],'2')
        word=word.replace(can[2],'3')
        word=word.replace(can[3],'4')
        if word.isdigit(): 
            conti=True
            for i in range(len(word)-1):
                if word[i]==word[i+1]:
                    conti=False
                    break
            if conti: answer+=1
    return answer

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.java
import java.util.*;
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        for(String word:babbling){
            word=word.replace("aya","0");
            word=word.replace("ye","1");
            word=word.replace("woo","2");
            word=word.replace("ma","3");      
            boolean b=true;
            for(int i=0;i<word.length();i++){
                
                if(!Character.isDigit(word.charAt(i))){
                    b=false;
                    break;
                }
                if(i>0&&word.charAt(i-1)==word.charAt(i)){
                    b=false;
                    break;
                }
                
            }
            if(b) answer++;
            
        }
        
       
        return answer;
    }
}